app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -270,8 +272,21 @@ def pod_full_name
     "#{options[:pod_namespace]}/#{options[:pod_name]}"
   end
 
+  def inspector_admin_secret
+    kubeclient = kubernetes_client
+    begin
+      inspector_sa = kubeclient.get_service_account(IMAGE_INSPECTOR_SA, INSPECTOR_NAMESPACE)
+      # TODO: support multiple imagePullSecrets. This depends on image-inspector support
+      return inspector_sa.try(:imagePullSecrets).to_a[0].try(:name)
+    rescue KubeException => e
+      raise e unless e.error_code == 404
+      _log.warn("Service Account #{IMAGE_INSPECTOR_SA} does not exist.")
+      return nil
```

Comment:
Minor: @enoodle move this outside of the `rescue` block to make sure that we always return `nil` (also clearer to the reader).